exosphere-shared/src/docker-helper.ls

```diff
@@ -1,24 +1,26 @@
 require! {
   \prelude-ls : {any, head, map}
   'dockerode' : Docker
-  'wait' : {wait}
+  'text-stream-search' : TextStreamSearch
 }
 
 docker = new Docker
 
 # Helper class used to manage Docker processes not started by docker-compose
 class DockerHelper
 
-  @start-container = ({Image, name, HostConfig}, done) ->
+  @start-container = ({Image, name, HostConfig, online-text}, done) ->
     DockerHelper.list-running-containers (err, running-containers) ->
       | err                              => done err
       | running-containers.includes name => done!
       | otherwise                        =>
         docker.create-container {Image, name, HostConfig}, (err, container) -> 
           | err => done err
-          container.start (err, data) ->
-            | err => done err
-            wait 2_000, done
+          container.attach {stream: true, stdout: true, stederr: true}, (err, stream) ->
+            stream = new TextStreamSearch stream    
```

Comment:
Is it possible then to miss data that's been passed through the stream between calling `container.start` and when `TextStreamSearch` is constructed in the callback of `container.start`? I figured attaching to the stream before starting the container would ensure capturing all output from the process.